packages/cubejs-materialize-driver/src/MaterializeDriver.ts

```diff
@@ -49,6 +49,29 @@ export class MaterializeDriver extends PostgresDriver {
     return BaseDriver.prototype.uploadTableWithIndexes.bind(this)(table, columns, tableData, indexesSql, [], null);
   }
 
+  /**
+   * Materialize quereable schema
+   * Returns materialized sources, materialized views, and tables
+   * @returns {string} schemaQuery
+   */
+  public informationSchemaQuery(): string {
+    const materializationFilter = `
+        table_name IN (
+          SELECT name
+          FROM mz_catalog.mz_sources
+          WHERE mz_internal.mz_is_materialized(id)
+          UNION
+          SELECT name
+          FROM mz_catalog.mz_views
+          WHERE mz_internal.mz_is_materialized(id)
+          UNION
+          SELECT t.name
+          FROM mz_catalog.mz_tables t
+        )`;
+
+    return `${super.informationSchemaQuery()} AND ${materializationFilter}`;
```

Comment:
Are you sure you are happy with `${super.informationSchemaQuery()}`? I'm going to merge tomorrow provided you are indeed OK with that.

It currently reads:

```
return `
      SELECT columns.column_name as ${this.quoteIdentifier('column_name')},
             columns.table_name as ${this.quoteIdentifier('table_name')},
             columns.table_schema as ${this.quoteIdentifier('table_schema')},
             columns.data_type as ${this.quoteIdentifier('data_type')}
      FROM information_schema.columns
      WHERE columns.table_schema NOT IN ('information_schema', 'mysql', 'performance_schema', 'sys', 'INFORMATION_SCHEMA')
   `;
```